Program.cs

```diff
@@ -620,7 +620,7 @@ private static void findSolutionPath()
             solutionPath = path;
         }
 
-        private static bool runCommand(string command, string args, bool useSolutionPath = true)
+        private static bool runCommand(string command, string args, bool useSolutionPath = true, bool setXAppImageVersion = false, string? version = null)

```

Comment:
I don't think the `bool` is required. You can just always set it.

Reply:
I removed the conditional entirely.